util_linux.c

```diff
@@ -10,10 +10,10 @@
 int32_t get_desktop_env(void) {
     const char *current_desktop = getenv("XDG_CURRENT_DESKTOP");  // Since 2012
     if (current_desktop) {
-        if (!strcasestr(current_desktop, "Unity") || !strcasestr(current_desktop, "GNOME") ||
-            !strcasestr(current_desktop, "XFCE")) {
+        if (strcasestr(current_desktop, "gnome") || strcasestr(current_desktop, "unity") ||
+            strcasestr(current_desktop, "xfce")) {
```

Comment:
Let keep old sting cases for the linker ability to eliminate duplicate strings.

Reply:
There are strings below this that are all lowercase, so lowercase would be better for string pooling.

Reply:
Take a look at here, since it need to be updated
```bash
ubuntu@ubuntu-2204:~$ echo $DESKTOP_SESSION
ubuntu
ubuntu@ubuntu-2204:~$ echo $XDG_CURRENT_DESKTOP
ubuntu:GNOME
```